stable-diffusion/anaconda-project.yml

```diff
@@ -0,0 +1,120 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+
+# required: name MUST match the directory name.
+#           name must be of the form r'[a-z_]+'
+name: stable-diffusion
```

Comment:
The historic pattern on the project names is to use `_` instead of `-`. You will have to update this name, the directory name, the notebook name and the thumbnail name.

Reply:
Yep - saw the updated docs. That's usually best practice for *nix - not sure why/how I missed that. I'll fix. Thanks